extensions/data-transfer/portability-data-transfer-imgur/src/test/java/org/datatransferproject/transfer/imgur/photos/ImgurPhotoExporterTest.java

```diff
@@ -100,7 +108,23 @@ public void setUp() throws IOException {
     server = new MockWebServer();
     server.start();
     exporter =
-        new ImgurPhotosExporter(monitor, client, mapper, jobStore, server.url("").toString());
+        new ImgurPhotosExporter(monitor, client, mapper, jobStore, server.url("").toString(),
+            (urlString) -> {
+              URL u = null;
+              try {
+                u = spy(new URL(urlString));
+              } catch (MalformedURLException e) {
```

Comment:
(nit; up to you if you feel like tackling this right now)

i see why the try/catch in the real code, but probably not necessary in the test?

Reply:
The build failed without the try/catch, even with the call being in the test